app/services/task_service.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class TaskService
+  @count_per_page
+
+  def initialize(count_per_page: 10)
+    @count_per_page = count_per_page
+  end
+
+  def task_list(params: nil)
```

Comment:
Why is it `nil` by default? I would suggest to use an empty hash instead
```suggestion
  def task_list(params: {})
```